OnlineFlightSearchAPI/Repositories/AirportRepository.cs

```diff
@@ -1,22 +1,22 @@
-﻿using System.Linq;
-using OnlineFlightSearchAPI.Models;
+﻿using OnlineFlightSearchAPI.Models;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace OnlineFlightSearchAPI.Repositories
 {
     public class AirportRepository : IAirportRepository
     {
         private readonly List<AirportDetail> airportDetails = new List<AirportDetail>();
 
-        public bool IsAirportValid(string airportCode)
+        public List<AirportDetail> FetchAirportDetail(string airportCode)
```

Comment:
Avoid returning concrete class.
Also, prefer returning least derived type. That gives more freedom if implementation is changed later.